spec/mongoid_spec.rb

```diff
@@ -409,7 +409,7 @@ def extension_white_list
     end
 
     after do
-      FileUtils.rm_rf(file_path("uploads"))
+      FileUtils.rm_rf(public_path("uploads"))
```

Comment:
It doesn't fix anything but it seems to me like it's `public/uploads` we should remove not `fixtures/uploads`.